src/app/components/List/EBookList.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import appConfig from '../../../../appConfig';
+
+const ebookUrl = appConfig.ereader[process.env.APP_ENV];
```

Comment:
If it wasn't for webpack, this wouldn't work. I'd prefer if this (and the other instances where `process.env....` are called) are only called once at the top level (`app.jsx` I believe) and then the value passed down to each component that needs it. Maybe through context?

Reply:
Setting this up as a prop in Application.jsx and passing along to children. Could be more efficient it seems to me.